### DISCUSSION TITLE
Provide PDF invoice for annual subscribers

### DISCUSSION THREAD

**Discussion by bfencken** [plausible/analytics#684] [Anything Else]
Many bookkeepers like a printable invoice for paid subscription services. Perhaps I am not looking the right place? 

![image](https://user-images.githubusercontent.com/54144371/107069240-e2a6ad80-679e-11eb-863a-c0bc9d872153.png)

**Comment by metmarkosaric** [comment]
thanks for the suggestion @bfencken! we will consider it! at this moment, the invoices get sent automatically via email for each payment that you make. the emails are sent by our payment processor Paddle and are sent to the email address on the account. you receive them pretty much instantly after making a payment.

**Comment by metmarkosaric** [comment]
You can now download your invoices in your Plausible account. Log in to your Plausible account and in the top right menu, click on your account name and choose "Settings" to enter your account settings.

To download your invoices, find the "Invoices" section. You'll see the list of your invoices and you can download the individual invoice by clicking on the link button.

**Reply by EmanueleDB** [reply]
Hi, the invoice download button is not shown anymore. I can only 'view' my invoice but it is not possible to download it
